packages/kbn-optimizer/src/optimizer/watcher.ts

```diff
@@ -38,7 +38,6 @@ export class Watcher {
 
   private readonly watchpack = new Watchpack({
     aggregateTimeout: 0,
-    ignored: /node_modules\/([^\/]+[\/])*(?!package.json)([^\/]+)$/,
```

Comment:
We only watch the specific paths referenced by a bundle so this should be totally unnecessary, but had to be removed to allow us to watch the build output of the packages.